Pico8Emulator/unit/graphics/GraphicsUnit.cs

```diff
@@ -227,7 +242,15 @@ public class GraphicsUnit : Unit {
 			Emulator.Memory.DrawState.DrawColor = (byte) (col.Value & 0x0f);
 		}
 
-		private void Psett(int x, int y, byte? col = null) {
+        /// <summary>
+        /// A special kind of Pset only used in the Spr and Sspr functions.
+        /// It removes Fillp functionality and default color (DrawColor variable) update, 
+        /// since that should only work for functions like circ() and rect().
+        /// </summary>
+        /// <param name="x"> X screen position. </param>
+        /// <param name="y"> Y screen position. </param>
+        /// <param name="col"> Color to draw pixel. </param>
+		private void Spset(int x, int y, byte? col = null) {
```

Comment:
Should you use a multiline comment for docs? `/**` marks a start of documentation too.

Reply:
I only use it like this bc it's how it's generated by the plugin I use. For me it doesn't make a big difference, wdyt ?